docs/core/introduction.md

```diff
@@ -1,57 +1,379 @@
+
+Here are some examples of types defined in the runtime libraries:
+
+* Primitive types, such as <xref:System.Boolean?displayProperty=nameWithType> and <xref:System.Int32?displayProperty=nameWithType>.
+* Collections, such as <xref:System.Collections.Generic.List%601?displayProperty=nameWithType> and <xref:System.Collections.Generic.Dictionary%602?displayProperty=nameWithType>.
+* Data types, such as <xref:System.Data.DataSet?displayProperty=nameWithType> and <xref:System.Data.DataTable?displayProperty=nameWithType>.
+* Network utility types, such as <xref:System.Net.Http.HttpClient?displayProperty=nameWithType>.
+* [File and stream I/O](../standard/io/index.md) utility types, such as <xref:System.IO.FileStream?displayProperty=nameWithType> and <xref:System.IO.TextWriter?displayProperty=nameWithType>.
+* [Serialization](../standard/serialization/index.md) utility types, such as <xref:System.Text.Json.JsonSerializer?displayProperty=nameWithType> and <xref:System.Xml.Serialization.XmlSerializer?displayProperty=nameWithType>.
+* High-performance types, such as <xref:System.Span%601?displayProperty=nameWithType>, <xref:System.Numerics.Vector?displayProperty=nameWithType>, and [Pipelines](../standard/io/pipelines.md).
+
+For more information, see [Framework libraries](../standard/framework-libraries.md) and [the source code for the libraries](https://github.com/dotnet/runtime/tree/master/src/libraries).
+
+## Microsoft.Extensions libraries
+
+Libraries for some commonly used application functionality aren't included in the runtime libraries but are made available in NuGet packages, such as the following:
+
+| NuGet package | Documentation |
+|---------|---------|
+| [Microsoft.Extensions.Hosting](https://www.nuget.org/packages/Microsoft.Extensions.Hosting) | [Application lifetime management (Generic Host)](extensions/generic-host.md) |
+| [Microsoft.Extensions.DependencyInjection](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection) | [Dependency injection (DI)](extensions/dependency-injection.md)
+| [Microsoft.Extensions.Configuration](https://www.nuget.org/packages/Microsoft.Extensions.Configuration) | [Configuration](extensions/configuration.md) |
+| [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) | [Logging](/aspnet/core/fundamentals/logging) |
+
+For more information, see the [dotnet/extensions repository on GitHub](https://github.com/dotnet/extensions).
+
+## Data access
+
+.NET provides an Object/Relational Mapper (ORM) and a way to write SQL queries in code.
+
+### Entity Framework Core
+
+Entity Framework (EF) Core is an [open source](https://github.com/aspnet/EntityFrameworkCore) and cross platform data access technology that can serve as an ORM. EF Core lets you work with a database by referring to .NET objects in code. It reduces the amount of data-access code you would otherwise need to write and test. EF Core supports many database engines.
```

Comment:
```suggestion
Entity Framework (EF) Core is an [open source](https://github.com/aspnet/EntityFrameworkCore) and cross-platform data-access technology that can serve as an ORM. EF Core lets you work with a database by referring to .NET objects in code. It reduces the amount of data-access code you would otherwise need to write and test. EF Core supports many database engines.
```